src/MongoDB.Driver/FilterDefinitionBuilder.cs

```diff
@@ -796,6 +796,26 @@ public FilterDefinition<TDocument> In<TField>(Expression<Func<TDocument, TField>
             return In(new ExpressionFieldDefinition<TDocument, TField>(field), values);
         }
 
+        /// <summary>
+        /// Creates a json schema validator.
+        /// </summary>
+        /// <param name="schema">The json validation schema.</param>
+        /// <returns>An schema validator.</returns>
+        public FilterDefinition<TDocument> JsonSchema(string schema)
```

Comment:
I'm not sure if this overload is necessary or sufficient. 

It's  not necessary because it's easy for the app to call BsonDocument.parse itself.  
And it may not be sufficient because you may have some other document-like thing that you could encode to BSON with a serializer, e.g.

  JsonSchema(new MyJsonSchemaPoco(...))

What do you think, @rstam?

Reply:
I agree. Let's remove this overload.